src/DynamoCore/Core/PreferenceSettings.cs

```diff
@@ -21,7 +21,6 @@ namespace Dynamo
     public class PreferenceSettings : NotificationObject, IPreferences
```

Comment:
Parameter-less `Load` and `Save` methods on `PreferenceSettings` class assumes the XML file to be stored in certain location. The behavior was dependent on settings file path in `DynamoPathManager`, but since it is managed outside of `PreferenceSettings`, its outcome is in-deterministic. This change enforces the path is specified by the owner of `PreferenceSettings` object.